tck/src/main/java/org/eclipse/microprofile/lra/tck/TckLRATypeTests.java

```diff
@@ -93,145 +95,501 @@ public void after() {
         NONE, NOT_PRESENT, EQUALS, NOT_EQUALS
     }
 
+    // LRA annotations placed on the resource class (see LRATypeTckResource)
+
     @Test
     public void requiredWithLRA() {
-        resourceRequest(REQUIRED_PATH, true, 200, MethodLRACheck.EQUALS, false);
+        requiredWithLRA(TCK_LRA_TYPE_RESOURCE_PATH);
```

Comment:
This is odd, you have added an extra level of indirection for no particular reason. Before I could scan the tests and get a good feel for how they differ. Since the change has no bearing on the issue can we go back to how they were originally.

Reply:
the benefit is that all 3 resources are handled in the same way. I don't want to keep the original code because the same logic is used in all 3 cases.

Reply:
Here is an example of where the extra indirection just adds overhead:

```java
    @Test
    public void requiredWithLRA() {
        requiredWithLRA(TCK_LRA_TYPE_RESOURCE_PATH);
    }

    @Test
    public void requiredWithoutLRA() {
        requiredWithoutLRA(TCK_LRA_TYPE_RESOURCE_PATH);
    }

    private void requiredWithLRA(String rootPath) {
        resource Request(rootPath, REQUIRED_PATH, true, 200, MethodLRACheck.EQUALS, false);
    }

    private void requiredWithoutLRA(String rootPath) {
        resourceRequest(rootPath, REQUIRED_PATH, false, 200, MethodLRACheck.NOT_EQUALS, false);
    }
```

and all of the methods you changed do the same. The original code was clear and did not have this obfuscating indirection.